pkg/interface/config/webpack.dev.js

```diff
@@ -77,12 +99,12 @@ module.exports = {
     extensions: ['.js', '.ts', '.tsx']
   },
   devtool: 'inline-source-map',
-  // devServer: {
-  //   contentBase: path.join(__dirname, './'),
-  //   hot: true,
-  //   port: 9000,
-  //   historyApiFallback: true
-  // },
+  devServer: {
+    contentBase: path.join(__dirname, '../dist'),
+    hot: true,
```

Comment:
we can probably get real hot reload by proxying 404s to localhost:80 or wherever. This would kinda fall down on a multi ship dev env though.

Reply:
? maybe we can include that but with a comment to disable in multi-ship setups? I don't use multi-ship and it seems rare for FE development.

Reply:
I mean it's used for like, cross-ship chats and whatnot but usually the second ship doesn't receive the FE changes, it's just to model incoming stuff.